src/DeckEncoder.js

```diff
@@ -138,6 +138,6 @@ class DeckEncoder {
   }
 }
 
-DeckEncoder.MAX_KNOWN_VERSION = 2
+DeckEncoder.MAX_KNOWN_VERSION = 3
```

Comment:
According to [Riot's documentation](https://github.com/RiotGames/LoRDeckCodes#process) version 3 is not yet supported in the client and so this should still remain as version 2. If this is merged, it would be a breaking change.